CONTRIBUTING.md

```diff
@@ -55,25 +55,64 @@ From the repository root.
 Before making a PR, you'll need to do two things to get CI passing:
 
 1. Generate schema files for the contracts.
-2. Generate typescript interfaces from those schemas.
+2. Generate Typescript interfaces from those schemas.
 
-To generate schema files, run:
+### Generating schema files
 
+We generate JSON schema files for all of our contracts' query and
+execute messages. We use those schema files to generate Typescript
+code for interacting with those contracts via
+[ts-codegen](https://github.com/CosmWasm/ts-codegen). This generated
+Typescript code is then [used in the
+UI](https://github.com/DA0-DA0/dao-dao-ui/tree/40f3cbfe676a98bf7b9db7b646e74e5b2dae4502/packages/state/clients).
```

Comment:
"Used in the UI to...." Important to tell people why they need to do this.

Reply:
Great feedback. Added a blurb about frontend folks not needing to manually write query and execute messages.